### PR TITLE
feat: add gcp pubsub docs (e5.0)

### PR DESCRIPTION
**Pull request by thalesmg**

### CODE REVIEW

**Comment by thalesmg** [comment]
I'll need help with the ZH screenshots, as my browser does not render the characters properly...  :see_no_evil:

File: dir.yaml [hunk 1]
```diff
@@ -127,6 +127,10 @@
         - data-integration/data-bridges
         - data-integration/data-bridge-webhook
         - data-integration/data-bridge-mqtt
+        - title_en: GCP PubSub Bridge
+          title_cn: TODO
```
Comment (ieQu1) [inline dir.yaml]: TODO?

**Review by ieQu1 — DISMISSED** [review]
Ok for the English part

File: dir.yaml (hunk 1 above)
Comment (thalesmg) [reply dir.yaml]: Oops :see_no_evil:

File: zh_CN/data-integration/data-bridge-gcp-pubsub.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+应用。
+
+下面的步骤将指导你使用这个配置。
+
+## 设置
+
+在配置GCP PubSub之前，必须先在GCP方面进行一些配置步骤
+在EMQX上配置GCP PubSub桥。
+
+1. 1. 创建一个[服务
```
Comment (qzhuyan) [inline zh_CN/data-integration/data-bridge-gcp-pubsub.md]:
```suggestion
1.  创建一个[服务
```

File: zh_CN/data-integration/data-bridge-gcp-pubsub.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+   账户](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount)
+   在你的GCP账户中。 确保该服务账户有
+   权限，至少可以向感兴趣的主题发布消息。
+2. 2. 为该账户创建一个服务账户密钥，并以JSON格式下载它。
```
Comment (qzhuyan) [inline zh_CN/data-integration/data-bridge-gcp-pubsub.md]: same here

File: zh_CN/data-integration/data-bridge-gcp-pubsub.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+   JSON格式。
+3. 3. 创建一个PubSub主题（记住，服务帐户必须有
+   权限来发布到该主题）。
+
+## GCP PubSub桥通过仪表盘使用的例子
+
+进入EMQX仪表板，选择左边菜单上的_"数据集成"_项目，然后选择_"数据桥"。
```
Comment (qzhuyan) [inline zh_CN/data-integration/data-bridge-gcp-pubsub.md]:
```suggestion
进入EMQX仪表板，选择左边菜单上的_"数据集成"_项目，然后选择_"数据桥"_。
```

File: zh_CN/data-integration/data-bridge-gcp-pubsub.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+在左边的菜单上选择_"数据集成"_，然后选择_"数据桥"_。 然后，点击_"+创建"_。
+
+在对话框中，选择_Google PubSub_桥类型，点击下一步。
+填写该桥的必要字段（这些字段标有
+星号）。 _payload模板_字段，如果留空，将编码
+所有来自MQTT消息的可见输入都使用JSON格式，例如
+clientid, topic, payload, 等等。 否则，它将使用用户定义的
+模板，用MQTT上下文中的相应值来填充`${变量_名称}`形式的占位符。
+对应的MQTT上下文中的值。 例如，`${topic}`将被替换为`my/topic}。
```
Comment (qzhuyan) [inline zh_CN/data-integration/data-bridge-gcp-pubsub.md]:
```suggestion
对应的MQTT上下文中的值。 例如，`${topic}`
```

File: zh_CN/data-integration/data-bridge-gcp-pubsub.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+将被替换为`my/topic`，如果这是MQTT消息的主题。
+
+在_GCP服务账户凭证_字段中，上传你的JSON格式的服务账户凭证。
+帐户凭证的JSON格式，你在设置步骤中导出的。
+
+![创建一个GCP PubSub桥](./assets/gcp_pubsub/setup1.png)
```
Comment (qzhuyan) [inline zh_CN/data-integration/data-bridge-gcp-pubsub.md:41]: nit, the png files are in English and the button names here in this doc are in Chinese.

File: zh_CN/data-integration/data-bridge-gcp-pubsub.md (hunk 3 above)
Comment (thalesmg) [reply zh_CN/data-integration/data-bridge-gcp-pubsub.md]:
```suggestion
2. 为该账户创建一个服务账户密钥，并以JSON格式下载它。
```

File: zh_CN/data-integration/data-bridge-gcp-pubsub.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+在配置GCP PubSub之前，必须先在GCP方面进行一些配置步骤
+在EMQX上配置GCP PubSub桥。
+
+1. 1. 创建一个[服务
+   账户](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount)
+   在你的GCP账户中。 确保该服务账户有
+   权限，至少可以向感兴趣的主题发布消息。
+2. 2. 为该账户创建一个服务账户密钥，并以JSON格式下载它。
+   JSON格式。
+3. 3. 创建一个PubSub主题（记住，服务帐户必须有
```
Comment (thalesmg) [inline zh_CN/data-integration/data-bridge-gcp-pubsub.md]:
```suggestion
3. 创建一个PubSub主题（记住，服务帐户必须有
```

File: zh_CN/data-integration/data-bridge-gcp-pubsub.md (hunk 6 above)
Comment (thalesmg) [reply zh_CN/data-integration/data-bridge-gcp-pubsub.md:41]: Yes, I'll need help with the ZH screenshots, as my browser does not render the characters properly... :see_no_evil:

Comment (zhongwencool) [reply zh_CN/data-integration/data-bridge-gcp-pubsub.md:41]: done

**Review by zhongwencool — APPROVED** [review]

**Comment by thalesmg** [comment]
@zhongwencool Thank you for the screenshots!  :beers:

**Pull request merged**